src/core_plugins/metrics/server/lib/calculate_indices.js

```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash';
```

Comment:
seems this file (and few others in this folder?) is only used from vis_data folder ... so maybe it belongs there ?